papaparse.js

```diff
@@ -682,7 +682,10 @@ License: MIT
 				return;
 			}
 
-			this._start += xhr.responseText.length;
+			// The length of the responseText doesn't corresponds to the actual size we requested due to potential encoding
```

Comment:
It can be simplified to: "Use chunckSize as it may be a diference on reponse lentgh due to characters with more than 1 byte.